azimuth/routers/v1/export.py

```diff
@@ -71,6 +71,20 @@ def export_dataset(
     return FileResponse(path=pt, filename=filename)
 
 
+@router.get(
+    "/dataset_splits/{dataset_split_name}/proposed_actions",
+    summary="Export proposed actions as csv.",
+    description="Export proposed actions to a CSV file and returns it.",
+    tags=TAGS,
+    response_class=FileResponse,
+)
+def export_proposed_actions(
+    dataset_split_manager: DatasetSplitManager = Depends(get_dataset_split_manager),
+) -> FileResponse:
+    pt = dataset_split_manager.save_proposed_actions_to_csv()
+    return FileResponse(path=pt, filename=os.path.basename(pt))
```

Comment:
I'm sorry, detail, but could we spell `path`? I think "pt" is more often used for "point".
```suggestion
    path = dataset_split_manager.save_proposed_actions_to_csv()
    return FileResponse(path=path, filename=os.path.basename(path))
```

Reply:
No problem, I'll replace a few others in this file.